portfolio_gallery/backend/src/schema/resolvers-types.ts

```diff
@@ -0,0 +1,131 @@
+import { GraphQLResolveInfo } from 'graphql';
```

Comment:
[nitpick] - it's just a common practice that has nothing to do with performance or anything, it's common the leave a space between import and others line below;
```suggestion
import { GraphQLResolveInfo } from 'graphql';

export type Maybe<T> = T | null;
export type InputMaybe<T> = Maybe<T>;
```

Reply:
Thank you Kei, OK I think it does not take a long time. I will try to set format rules and set it on commit!
#45